zipkin-lens/package.json

```diff
@@ -91,7 +90,6 @@
     "lint": "eslint \"src/**/*.{js,jsx,ts,tsx}\"",
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "postbuild": "mvy -f build target/classes/zipkin-lens",
```

Comment:
this was easier than the maven alternative, but it is abandoned so nagging npm audit